packages/integrations/useProgressBar/index.ts

```diff
@@ -0,0 +1,28 @@
+import progress, { NProgressOptions } from 'nprogress'
+import { MaybeRef, tryOnUnmounted } from '@vueuse/shared'
+import { ref, watch, isRef } from 'vue-demi'
+
+/**
+ * Reactive progress bar.
+ *
+ * @see   {@link https://vueuse.js.org/useProgressBar}
+ * @param newIcon
+ * @param options
+ */
+export function useProgressBar(
+  current: MaybeRef<boolean | null | undefined> = null,
```

Comment:
Thinking we should use numbers to set the progress.

https://github.com/rstacruz/nprogress#advanced-usage

Reply:
```js {13}
export function useNProgress(
  currentProgress: MaybeRef<number | null | undefined> = null,
  options?: NProgressOptions | undefined,
) {
  const progress = isRef(currentProgress) ? currentProgress : ref(null)
  const isLoading = ref(false)

  if (options)
    nprogress.configure(options)

  watch(isLoading, active => active ? nprogress.start() : nprogress.done())

  watch(progress, (n) => {
    if (typeof n === 'number')
      nprogress.set(n)
  })

  tryOnUnmounted(nprogress.remove)

  return {
    isLoading,
    progress,
    start: nprogress.start,
    done: nprogress.done,
    remove: nprogress.remove,
  }
}
```

What do you think?